canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -756,6 +757,40 @@ export default class PropertiesController {
 		this.propertiesStore.clearSelectedRows(propertyId);
 	}
 
+	/**
+	 * Disable table row move buttons for all propertyIds in given array
+	 * @param propertyIds Array of propertyIds
+	 *
+	 */
+	setDisableRowMoveButtons(propertyIds) {
+		// Verify that input is an array of objects
+		if (Array.isArray(propertyIds)) {
+			this.propertiesStore.setDisableRowMoveButtons(propertyIds);
+		}
+	}
+
+	/**
+	* Returns array of propertyIds for which row move buttons will be disabled
+	*	@return Array of propertyIds
+	*/
+	getDisableRowMoveButtons() {
+		return this.propertiesStore.getDisableRowMoveButtons();
+	}
+
+	/**
+	 * Check if row move buttons should be disabled for given propertyId
+	 * @param propertyId  The unique property identifier
+	 * @return boolean
+	 */
+	checkIfDisableRowMoveButtons(propertyId) {
+		const propertyIds = this.getDisableRowMoveButtons();
+		return (
+			Array.isArray(propertyIds)
+				? propertyIds.some((el) => getDataId(el) === getDataId(propertyId))
```

Comment:
I wonder if using https://lodash.com/docs/4.17.15#isEqual would work better here?  Then you don't have to worry about converting to a string.